render-only-sample/roscommon/Vc4Hw.h

```diff
@@ -990,6 +990,21 @@ typedef enum _VC4TextureMinFilter
 const UINT VC4_BINNING_TILE_PIXELS  = 64;
 
 //
+// Constants related to tiled textures. Values are measured in bytes.
+//
+
+const UINT VC4_1KBLOCK_WIDTH    = 64;
+const UINT VC4_1KBLOCK_HEIGHT   = 16;
+const UINT VC4_1KBLOCK_SIZE     = (VC4_1KBLOCK_WIDTH * VC4_1KBLOCK_HEIGHT);
+
+const UINT VC4_4KBLOCK_WIDTH    = 128;
+const UINT VC4_4KBLOCK_HEIGHT   = 32;
+const UINT VC4_4KBLOCK_SIZE     = (VC4_4KBLOCK_WIDTH * VC4_4KBLOCK_HEIGHT);
+
+const UINT VC4_MICRO_TILE_SIZE  = 64;
+const UINT VC4_MICRO_TILE_WIDTH = 16;
+
+//
 // VC4 bus address alias
```

Comment:
Hi, Marek:

    Your change looks good overall. So these are all small things.

    Can you please change 4KBLOCK to 4KB_TILE and 1KBLOCK to 1KB_SUB_TILE to be consistent with the VC4 spec ?

    I would suggest that we define VC4_1KB_SUB_TILE_WIDTH as 16 in pixels so that it is consistent with VC4_1KB_SUB_TILE_HEIGHT, please feel free to define another VC4_1KB_SUB_TILE_WIDTH_BYTES as 64 or calculate it on the fly. Can you please change VC4_1KB_SUB_TILE_SIZE to VC4_1KB_SUB_TILE_SIZE_BYTES and simply define it as 1024 ? (It will not change with formats)

    Same suggestion for the 4K tile and macro tile.

                                           Thanks, 2/9/2016